framework/Steps/Step.py

```diff
@@ -103,7 +103,7 @@ class cls.
     inputSpecification.addParam("re-seeding", InputTypes.StringType, descr=r"""
               this optional
               attribute could be used to control the seeding of the random number generator (RNG).
-              If inputted, the RNG can be reseeded. The value of this attribute
+              If inputted, the RNG can be re-seeded. The value of this attribute
```

Comment:
one of the unit test is failed due to the updated test, I think you need to regold that test